.github/workflows/release.yml

```diff
@@ -44,3 +44,27 @@ jobs:
     - uses: actions/checkout@master
     - name: Update new version in krew-index
       uses: rajatjindal/krew-release-bot@v0.0.40
+  publish-image:
+    name: publish images
+    # prevent job running from forked repository, otherwise
+    # 1. running on the forked repository would fail as missing necessary secret.
+    # 2. running on the forked repository would use unnecessary GitHub Action time.
+    if: ${{ github.repository == 'karmada-io/karmada' }}
+    runs-on: ubuntu-18.04
+    steps:
+    - name: checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Just a question, it defaults to check out the code from the master branch, right?
What if we cut a release from the release branch? like cut `v1.0.2` from branch `release-1.0`?